sds_data_manager/lambda_code/IAlirtCode/ialirt_ingest.py

```diff
@@ -19,6 +27,93 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+KERNELS = {
+    "ephemeris_predicted",
+    "ephemeris_90days",
+    "planetary_ephemeris",
+    "spacecraft_clock",
+    "leapseconds",
+    "imap_frames",
+    "science_frames",
+}
+EFS_BASE_PATH = Path("/mnt/data")
+
+
+def get_ephemeris_time():
+    """Get the current ephemeris time in seconds since J2000.
+
+    Returns
+    -------
+    et: float
+        The current ephemeris time in seconds since J2000.
+    """
+    now = datetime.now(timezone.utc)
+    j2000 = datetime(2000, 1, 1, 12, 0, 0, tzinfo=timezone.utc)
```

Comment:
I'm not sure what the needed accuracy of this function is, but it is not quite correct.
This is not the correct epoch for J2000.
From the imap_processing.spice.time.py file:
```
# The UTC string was generated by:
# >>> spiceypy.et2utc(spiceypy.unitim(0, "TT", "ET"), "ISOC", 9)
TTJ2000_EPOCH = np.datetime64("2000-01-01T11:58:55.816", "ns")
```
Furthermore, ET is TDB which has a non-fixed offset from TT. The safest approach would be to use spiceypy for this calculation but perhaps that precision is not needed?

Reply:
I ran into the chicken/egg situation here. I needed approximate timestamps to furnish the most recent kernels. But to furnish the most recent kernels I needed the leapseconds kernel to use your library: str_to_et. So I ended up copying what dependency.py does:

# TODO: remove this once Bryan changes takes in 'yyyymmdd' format
            def yyyymmdd_to_seconds_since_j2000(
                date_str: str, add_24_hrs=False
            ) -> float:
                # Parse input date string
                dt = datetime.strptime(date_str, "%Y%m%d").replace(tzinfo=timezone.utc)
                if add_24_hrs:
                    dt += timedelta(hours=24)
                # Define J2000 epoch: 2000-01-01T12:00:00 UTC
                j2000 = datetime(2000, 1, 1, 12, 0, 0, tzinfo=timezone.utc)

                # Compute seconds difference
                delta = dt - j2000
                return delta.total_seconds()

I can add a TODO to change when Bryan changes takes in 'yyyymmdd' format. Otherwise I think this might be the best that I can do for now.

Reply:
Ok, I would still suggest fixing the J2000 epoch to reduce the error.
```suggestion
    j2000 = datetime(2000, 1, 1, 11, 58, 55.816, tzinfo=timezone.utc)
```